portfolio/src/main/webapp/style.css

```diff
@@ -3,7 +3,42 @@
   margin-right: auto;
   width: 650px;
 }
+#sidebar a{
```

Comment:
Have a space before all the open brackets so it's easier to read, and also an empty line between each selector. (ie line 6 should be an empty line, and #sidebar a { will start on line 7).